pkg/subscriptions/subscriptions.html

```diff
@@ -0,0 +1,157 @@
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+    <div class="modal" id="subscriptions-register-dialog" tabindex="-1" role="dialog" data-backdrop="static">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h4 class="modal-title">Register system</h4>
+          </div>
+          <div class="modal-body">
+            <table class="cockpit-form-table">
+              <tr>
+                <td translatable="yes">URL</td>
+                <td>
+                  <input class="form-control" id="subscription-register-url" type="text"/>
+                </td>
+              </tr>
+              <tr>
+                <td translatable="yes">User name</td>
+                <td>
+                  <input class="form-control" id="subscription-register-username" type="text"/>
+                </td>
+              </tr>
+              <tr>
+                <td translatable="yes">Password</td>
+                <td>
+                  <input class="form-control" type="password" id="subscription-register-password" value=""/>
+                </td>
+              </tr>
+            </table>
+          </div>
+          <div class="modal-footer">
+            <button class="btn btn-default" translatable="yes" data-dismiss="modal">
+              Cancel
+            </button>
+            <button class="btn btn-primary" id="account-register-start" translatable="yes">
+              Register
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <script id="subscription-template" type="x-template/mustache">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span>Installed Product ({{product_name}})</span>
+        </div>
+        <div class="panel-body">
+          <table class="cockpit-form-table">
+            <tbody>
+              <tr><td>Product name</td><td><span>{{product_name}}</span></td></tr>
+              <tr><td>Product ID</td><td><span>{{product_id}}</span></td></tr>
+              <tr><td>Version</td><td><span>{{version}}</span></td></tr>
+              <tr><td>Architecture</td><td><span>{{arch}}</span></td></tr>
+              <tr><td>Status</td><td><span>{{status}}</span></td></tr>
+              <tr><td>Starts</td><td><span>{{starts}}</span></td></tr>
+              <tr><td>Ends</td><td><span>{{ends}}</span></td></tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </script>
+
+    <div id="subscriptions-subscribed" class="container-fluid" hidden>
+      <div class="panel panel-default" id="subscription-system-info" hidden>
+        <div class="panel-heading">
+          <span>System</span>
+        </div>
+        <div class="panel-body">
+          <table class="cockpit-form-table">
+            <tbody>
+              <tr><td>System identity</td><td><span id="subscription-system-identity"></span></td></tr>
```

Comment:
> What are these cells used for?

@andreasn We can get additional system info (such as registration ID) from subscription manager. I thought it might be useful, but it's not crucial. What do you think?